robot.py

```diff
@@ -123,20 +136,58 @@ def perpendicular_line(self, line_distance):
         self.tank.turn(turn * -1)
         while not stop_on_black(left_cs, right_cs):
             self.tank.drive(100, 0)
-        self.tank.stop()
+        self.brake()
     
-    def steering(self, speed, sharpness, millies):
+
+    def steering(self, speed, sharpness, distance):
+        ''' Makes the robot able to turn and go forward
+
+        Args:
+
+        '''
         self.tank.reset()
-        big = millies
-        small = -1 * millies
+        big = distance
+        small = -1 * distance
         while self.tank.distance() >= small and self.tank.distance() <= big:
             self.tank.drive(speed, sharpness)
-        self.tank.stop()   
+        self.brake()   
     
+
     def steering_angle(self, speed, sharpness, angle):
+        ''' Steers the robot with a certain angle and speed
+
+        Args:
+            speed: the speed of the steering
+            sharpness: the sharpness or urgency
+            angle: the angle the robot is aiming to stop at
+        '''
         big = angle + 1
         small = angle - 1
         while self.gyro.angle() <= small or self.gyro.angle() >= big:
             self.tank.drive(speed, sharpness)
-        self.tank.stop()
-    
+        self.brake()
+
+
+    def straight_distance(self, distance, speed=150):
+        ''' Goes straight for a certain distance
+
+        Args:
+            speed: the speed of the robot
+            distance: the distance the robot should go
+                straight
```

Comment:
nit: this single word can on the previous line to make the function vertical length shorter.